packages/runtime-dom/src/modules/attrs.ts

```diff
@@ -1,11 +1,12 @@
-const xlinkNS = 'http://www.w3.org/1999/xlink'
+const XLINK_NS = 'http://www.w3.org/1999/xlink'
+const XLINK_ATTR_PREFIX = 'xlink:'
 
 function isXlink(name: string): boolean {
-  return name.charAt(5) === ':' && name.slice(0, 5) === 'xlink'
+  return name.startsWith(XLINK_ATTR_PREFIX)
```

Comment:
I think previously was faster than this change, since if the char[5] !== ':' it will not do the slice.

Reply:
I think `startsWith` is more readable, we don't want 

```js
name.charAt(0) === 'x' &&
name.charAt(1) === 'l' && 
name.charAt(2) === 'i' &&
name.charAt(3) === 'n' &&
name.charAt(4) === 'k' &&
name.charAt(5) === ':'
```

right?

Reply:
```suggestion
  return name.charAt(5) === ':' && name.indexOf('xlink') === 0
```
Since `indexOf` is faster than startsWith

but using charAt will be the fastest of them all

Reply:
It's random attribute, there is no difference with

```js
name.charAt(0) === 'x' && name.indexOf('link:') === 1
```

so, no thanks.
I can't commit this suggestion, I'd rather not touch it